cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/endtoend/MiscTests.java

```diff
@@ -0,0 +1,483 @@
+                         Tester.builder(TestSchema.builder(bridge)
+                                                  .withPartitionKey("pk", bridge.uuid())
+                                                  .withClusteringKey("ck", bridge.aInt())
+                                                  .withColumn("a", bridge.bigint())
+                                                  .withColumn("b", bridge.text())
+                                                  .withColumn("c", bridge.blob())
+                                                  .withBlobSize(400000)  // Override blob size to write large blobs that we can skip
+                                                  .withCompression(enableCompression))
+                               // Test with LZ4 enabled & disabled
+                               .withColumns("pk", "ck", "a")  // Partition/clustering keys are always required
+                               .withExpectedRowCountPerSSTable(Tester.DEFAULT_NUM_ROWS)
+                               .withStatsClass(MiscTests.class.getName() + ".STATS")  // Override stats so we can count bytes skipped
+                               .withCheck(dataset -> {
+                                   MiscTests.resetStats();
+                                   List<Row> rows = dataset.collectAsList();
+                                   assertThat(rows.isEmpty()).isFalse();
+                                   for (Row row : rows)
+                                   {
+                                       assertThat(row.schema().getFieldIndex("pk").isDefined()).isTrue();
+                                       assertThat(row.schema().getFieldIndex("ck").isDefined()).isTrue();
+                                       assertThat(row.schema().getFieldIndex("a").isDefined()).isTrue();
+                                       assertThat(row.schema().getFieldIndex("b").isDefined()).isFalse();
+                                       assertThat(row.schema().getFieldIndex("c").isDefined()).isFalse();
+                                       assertThat(row.length()).isEqualTo(3);
+                                       assertThat(row.get(0) instanceof String).isTrue();
+                                       assertThat(row.get(1) instanceof Integer).isTrue();
+                                       assertThat(row.get(2) instanceof Long).isTrue();
+                                   }
+                                   // TODO(c4c5): Why statistics are zero for C* 5 bridge?
+                                   if (bridge.getVersion().versionNumber() < 5)
+                                   {
+                                       assertThat(skippedRawBytes.get() > 50_000_000).isTrue();
+                                       assertThat(skippedInputStreamBytes.get() > 2_500_000).isTrue();
+                                       assertThat(skippedRangeBytes.get() > 5_000_000).isTrue();
+                                   }
```

Comment:
The TODO is already addressed. 

```suggestion
                                   assertThat(skippedRawBytes.get() > 50_000_000).isTrue();
                                   assertThat(skippedInputStreamBytes.get() > 2_500_000).isTrue();
                                   assertThat(skippedRangeBytes.get() > 5_000_000).isTrue();
```

Reply:
Done.